### PR TITLE
Implement API route to add new "Education" and return all "Educations"

### LINKED ISSUE #19: Implement API route to add new "Education"
On the `/resume/education` route, you should:

- [ ] Return the index position of the new Education in the Education list when a POST request is made

### PR DESCRIPTION
**Pull request by Paulooh007**
## Fixes
Fixes #19, #18 and #17 by @wrussell1999 

## Description
Here's what's included in this PR:
- [x] Return the index position of the new Education in the Education list when a POST request is made
- [x] Return the Education list when a GET request is made
- [x] Return a specific Education when a GET request is made with an index as an input

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,5 @@
 .venv
 .pytest_cache/
 __pycache__
-.vscode
\ No newline at end of file
+.vscode
+.DS_Store
```
Comment (fissoreg) [inline .gitignore:6]: This is not related to the current PR, it would be great to have it in its own PR so we can keep track of the changes to `.gitignore`.

File: app.py [hunk 2]
```diff
@@ -58,10 +58,20 @@ def education():
     Handles education requests
     '''
     if request.method == 'GET':
-        return jsonify({})
+        index = request.args.get('index')
+        if index is not None and index.isdigit():
+            index = int(index)
+            if 0 <= index < len(data["education"]):
+                return jsonify(data["education"][index])
```
Comment (fissoreg) [inline app.py:65]: What if the index is invalid? Can we return a meaningful error?

**Review by fissoreg — CHANGES_REQUESTED** [review]
Thank you @Paulooh007 for the PR, it looks great! :)
I left a couple of comments for you to review, let me know what you think.

File: .gitignore (hunk 1 above)
Comment (Paulooh007) [reply .gitignore:6]: Ok, No problem.
I'll revert this change.

File: app.py (hunk 2 above)
Comment (Paulooh007) [reply app.py:65]: Yes, i agree.
I'll can do this by using the `abort` function to return a `404 Not Found response` with a custom error message when index is invalid, as suggested.

**Comment by fissoreg** [comment]
Thank you @Paulooh007, merged! 👍
